src/commands/QueueCommand.ts

```diff
@@ -21,23 +21,45 @@ export class QueueCommand extends BaseCommand {
         if (Number(message.guild?.queue?.songs.size) > 12) {
             const indexes: string[] = this.chunk(songs!, 12);
             let index = 0;
+            const duration = message.guild?.queue?.songs.first()?.duration ?? 0;
             embed.setDescription(indexes[index]).setFooter(`Page ${index + 1} of ${indexes.length}`, "https://raw.githubusercontent.com/zhycorp/disc-11/main/.github/images/info.png");
             message.channel.send(embed).then(msg => {
-                msg.react("◀️").then(() => {
-                    msg.react("▶️").catch(e => this.client.logger.error("QUEUE_CMD_ERR:", e));
-                    msg.createReactionCollector((reaction, user) => reaction.emoji.name === "◀️" && user.id === message.author.id, { time: 80 * 1000 }).on("collect", () => {
-                        if (index === 0) return undefined;
-                        index--;
-                        embed.setDescription(indexes[index]).setFooter(`Page ${index + 1} of ${indexes.length}`, "https://raw.githubusercontent.com/zhycorp/disc-11/main/.github/images/info.png");
-                        msg.edit(embed).catch(e => this.client.logger.error("QUEUE_CMD_ERR:", e));
-                    });
-                    msg.createReactionCollector((reaction, user) => reaction.emoji.name === "▶️" && user.id === message.author.id, { time: 80 * 1000 }).on("collect", () => {
-                        if (index + 1 === indexes.length) return undefined;
-                        index++;
-                        embed.setDescription(indexes[index]).setFooter(`Page ${index + 1} of ${indexes.length}`, "https://raw.githubusercontent.com/zhycorp/disc-11/main/.github/images/info.png");
-                        msg.edit(embed).catch(e => this.client.logger.error("QUEUE_CMD_ERR:", e));
-                    });
-                }).catch(e => this.client.logger.error("QUEUE_CMD_ERR:", e));
+                void msg.react("◀️");
+                void msg.react("▶️");
+                const filter = (reaction: any, user: any) => user.id !== msg.client.user?.id;
```

Comment:
Missing return type on function

Reply:
According with discord.js docs `.createReactionCollector` is defined

```javascript
// Create a reaction collector
const filter = (reaction, user) => reaction.emoji.name === '👌' && user.id === 'someID';
const collector = message.createReactionCollector(filter, { time: 15000 });
collector.on('collect', r => console.log(`Collected ${r.emoji.name}`));
collector.on('end', collected => console.log(`Collected ${collected.size} items`));
```

https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=createReactionCollector